### DISCUSSION TITLE
Need to click "reconnect" every time I wake my computer up

### DISCUSSION THREAD

**Discussion by Aimsucks** [cgiesche/streamdeck-homeassistant#261] [Help and Support] [answered]
Waking the computer from sleep disconnects the plugin from my Home Assistant server and in order to get it working again, I need to go into every button and click "reconnect."

**Comment by cgiesche** [comment]
Hi,

this sound very strange to me. All buttons share one connection to home assistant. That means after you forced a reconnect on one button, all other buttons should be fixed, too. It may take a while until a new update from home-assistant arrives and the button comes back to live (state change of the entity of the button).

Besides that: There were issues like that before. On some computers the network interface seems to take longer to "wake up" from sleep than usual and so the plugin somehow gets lost. I will search for the issue and check if and how it was resolved.

**Reply by Aimsucks** [reply, marked as answer]
You can delete this discussion, I found an #247 fixed my issue.
